guides/doc-Tuning_Performance/master.adoc

```diff
@@ -50,7 +50,7 @@ include::common/modules/proc_manually-tuning-db-pool.adoc[leveloffset=+3]
 
 include::common/modules/con_apache-httpd-performance-tuning.adoc[leveloffset=+2]
 
-include::common/modules/proc_configuring-the-open-files-limit-for-apache-httpd.adoc[leveloffset=+3]
+include::common/modules/proc_tuning-apache-httpd-child-processes.adoc[leveloffset=+3]
```

Comment:
Should both of these includes be kept?

Reply:
sorry I didn't get your point.

Reply:
She means, either of the above files Okay? Or do you require both files (one is red, another green) to keep?

Reply:
Yes, last time I did exactly same. I misunderstood the last comment. Reverting it back.